src/pysdmx/model/message.py

```diff
@@ -36,6 +36,14 @@ class ActionType(Enum):
     Information = "information"
 
 
+SDMX_CSV_ACTION_MAPPER = {
```

Comment:
This is format-specific. Therefore, I would suggest to move this into its respective io package.

Reply:
Moving it to io.csv.sdmx20